walkthroughs/cdk-examples/app-mesh-hands-on/lib/stacks/ecs-services.ts

```diff
@@ -0,0 +1,110 @@
+import * as ecs from "aws-cdk-lib/aws-ecs";
+import { StackProps, Stack, CfnOutput } from "aws-cdk-lib";
+import { MeshStack } from "./mesh-components";
+import { AppMeshFargateService } from "../constructs/appmesh-fargate-service";
+import { EnvoySidecar } from "../constructs/envoy-sidecar";
+import { XrayContainer } from "../constructs/xray-container";
+import { ApplicationContainer } from "../constructs/application-container";
+
+export class EcsServicesStack extends Stack {
+  constructor(ms: MeshStack, id: string, props?: StackProps) {
```

Comment:
Would it be more readable to use more informative name like `meshStack` here?

Reply:
noted